### PR TITLE
appearance is no longer experimental + other fixes

### PR DESCRIPTION
**Pull request by zcorpan**
- With Chrome and Firefox shipping, it's no longer experimental
- Unprefixed appearance was added in Firefox 80
- Safari's support is still a partial implementation
- Move 'button' into &lt;compat-auto> per spec change https://github.com/w3c/csswg-drafts/issues/5174

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
  - https://wpt.fyi/results/css/css-ui/appearance-cssom-001.html?label=experimental&label=master&aligned
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any
  - https://github.com/mdn/browser-compat-data/pull/6505

### CODE REVIEW

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:40]: This seems right. Testing https://mdn-bcd-collector.appspot.com/tests/css/properties/appearance in Firefox 79 and 80, it starts returning true in 80.

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:137]: I haven't reviewed a change to `experimental` before, but here's what I found:
- https://github.com/mdn/browser-compat-data/issues/1528
- https://developer.mozilla.org/en-US/docs/MDN/Contribute/Guidelines/Conventions_definitions#Experimental
- https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#status-information

With this shipping in two implementations now, it's no longer experimental 👍

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:183]: I'll add the needs-release-note label since this is removing an entry.

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:221]: Can you say a bit about why this is considered partial? Is just because button was folded in? Notes would be helpful here, so one can tell when it should set to fully supported again, and so readers need not guess about what will work.

Comment (zcorpan) [reply css/properties/appearance.json:221]: Opera 15 corresponds to an early version of Chromium which doesn't follow the spec and doesn't support `auto` etc. However, whichever version of Opera that maps to Chromium 84 should have full support. https://help.opera.com/en/opera-version-history/ only shows up to Chromium 81.

Comment (zcorpan) [reply css/properties/appearance.json]: Fixed in 66b2ab0

I'm not sure how to explain the partial implementation in a short note. "Values can be applied to any element, rather than only widgets as per spec."?

**Comment by ddbeck** [comment]
@zcorpan I don't have anything to add to the review, but I saw this is your first PR to the repo. Welcome to BCD!

File: css/properties/appearance.json
Comment (foolip) [reply css/properties/appearance.json]: I usually check `browsers/opera.json` to find the matching release. But when preparing changing, you can also run something like `npm run mirror -- --modify=always opera css` and see what changes it makes.

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:189]: If any other review is confused by this, the name of the "production" really is `<compat-auto>` in CSS: https://drafts.csswg.org/css-ui-4/#typedef-appearance-compat-auto

Usually "compat" in this repo is something internal :)

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:251]: Can you add the non-`partial_implementation` entry here as well? If you run `npm run mirror -- --modify=always opera_android css` it will be among the resulting changes, but you may have to sift through some other changes. If they seem correct, feel free to throw them into this PR as well though.

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:263]: If you run `npm run mirror -- --modify=always samsunginternet_android css` you'll get a bunch of updates to apply. One of those ought to be here.

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:267]: Same story, can you run `npm run mirror -- --modify=always webview_android css`?

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:67]: This looks good, but it looks like some Firefox Android data is missing, if you run `npm run mirror -- --modify=always firefox_android css`.

**Review by foolip — COMMENTED** [review]
Thanks Simon, and welcome to BCD!

This looks good, and for any other first-time contributor I'd merge and follow up on the nits. But here, I might as well share the methods I'd use, if you continue contributing to BCD, which would be great :D

**Comment by zcorpan** [comment]
@foolip woah, magic 🧙‍♀️

OK fixed.

File: css/properties/appearance.json
Comment (ddbeck) [reply css/properties/appearance.json]: 👍 on adding notes for `partial_implementation`. That text is probably OK, but if you can point to an example or longer-form description of the problem, I can probably provide more detailed help on writing a note (it's definitely a challenging format for tech writing).

File: css/properties/appearance.json
Comment (foolip) [inline css/properties/appearance.json:107]:
```suggestion
                "version_added": "60"
```

`npm run mirror` didn't do the right thing here because ab53ef31eedfb0e36af82ebfbe8190bc694d1cf5 landed on master after you branched off.

**Review by foolip — APPROVED** [review]
LGTM % nit, which I'll try to apply myself.

File: css/properties/appearance.json
Comment (foolip) [reply css/properties/appearance.json:273]: This wasn't done, but I'm not going to block the PR on that happening. @AdaRoseCannon FYI that there will be some updating needed here. Some of these changes in these PR will only be possible to mirror when the next version of Samsung Internet, probably 13.1, is in BCD.

**Review by foolip — APPROVED** [review]
LGTM after applying that fix. I'll merge now.

**Pull request merged**
